JunitHomework/src/main/java/ro/sciit/junit/homework/Utility.java

```diff
@@ -0,0 +1,99 @@
+package ro.sciit.junit.homework;
+
+import java.lang.reflect.Array;
+import java.util.*;
+
+public class Utility {
+
+    private String input;
+    private List<String> numbersAsString = new ArrayList<String>(
+            Arrays.asList("zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"));
+    private List<String> numbersAsInt = new ArrayList<String>(
+            Arrays.asList("0", "1", "2", "3", "4", "5", "6", "7", "8", "9"));
+    private List<String> whitespaces = new ArrayList<String>(
+            Arrays.asList("", "//s", ","));
+    private List<String> userInputList = new ArrayList<String>();
+
+    /**
+     * method that removes white space from given input
+     * @return input as string with whitespaces removed
+     */
+    public String inputWithWhiteSpacesRemoved() {
+        //initializes user input
+        Scanner userInput = new Scanner(System.in);
+        System.out.println("Please enter a word or phrase: ");
+        // variable for storing the modified string
+        String stringWithoutWhiteSpaces = "";
+
+        //checks if scanner doesn't receive a break key, it continues to request user input
+        while (userInput.hasNextLine()) {
+            //transforms input to lower case letters
+            input = userInput.nextLine().toLowerCase();
+
+            //checks if input differs from null and contains some letters and is different from the letter q
+            if (input != null && !input.isEmpty() && !input.equals("q")) {
+                // input is added to userInputList
+                stringWithoutWhiteSpaces = input;
+                userInputList.add(stringWithoutWhiteSpaces);
+            } else if (input.equals("q")) {
+                //if the letter q is pressed, the program exits
+                break;
+            } else {
+                //if no input is entered, the program loops until something is entered
+                System.out.println("Nothing entered. Enter another word or type 'Q' to exit.");
+            }
+        }
+
+        //returns input as a string with whitespaces removed
+        return stringWithoutWhiteSpaces.replaceAll("\\s+", "");
+    }
+
+    /**
+     * method that replaces word written numbers with mathematical numbers
+     * @return input as string with numbers replaced
+     */
+    public String numberReplacer() {
+        Scanner userInput = new Scanner(System.in);
+        userInput = userInput.useDelimiter("\\n");
+        System.out.println("Please enter a word or phrase: ");
+        //variable that stores userInput
+        String splitInput = "";
```

Comment:
Here I would have split this method in 3 methods:
1) a method that reads from keyboard : public String readFromKeyboard(String endReadingKey);
2) a method that returns an array of words from a string: public List<String> sprintText(String rules);
3) a method that changes number as string in number as int:  public String numberReplacer(List<String> textInput);